meters/impl/abb.go

```diff
@@ -95,15 +112,26 @@ func (p *ABBProducer) Produce() (res []Operation) {
 		CurrentL1, CurrentL2, CurrentL3,
 		Power, PowerL1, PowerL2, PowerL3,
 	} {
-		res = append(res, p.snip32(op, 100))
+		res = append(res, p.snip32(op, 10))
```

Comment:
I couldn't test this, the docs only say res = "01". ok if this works for you?

Reply:
maybe they mean 0,1 and that makes it 10 :/

Reply:
we tested it and with this it shows the same as the display